core/src/main/java/org/apache/calcite/plan/volcano/VolcanoRuleMatch.java

```diff
@@ -145,16 +144,16 @@ private String computeDigest() {
       if (i > 0) {
         buf.append(", ");
       }
-      RelOptUtil.appendRelDescription(buf, rels[i]);
+      buf.append("#" + rels[i].getId());
     }
     buf.append("]");
```

Comment:
change to buf.append(']') to improvement perf slightly?

Reply:
done